godtools/App/Share/Data/UserCountersRepository/UserCountersRepository.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  UserCountersRepository.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 11/29/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class UserCountersRepository {
+    
+    private let api: UserCountersAPI
+    private let cache: RealmUserCountersCache
+    
+    init(api: UserCountersAPI, cache: RealmUserCountersCache) {
+        self.api = api
+        self.cache = cache
+    }
+    
+    func fetchRemoteUserCounters() -> AnyPublisher<[RealmUserCounter], URLResponseError> {
+        
+        return api.fetchUserCountersPublisher()
+            .flatMap { userCountersResponse in
+                
+                return self.cache.syncUserCounters(userCountersResponse.userCounters)
+                    .mapError { error in
+                        return URLResponseError.otherError(error: error)
+                    }
+                    .eraseToAnyPublisher()
+            }
+            .eraseToAnyPublisher()
+    }
+    
+    func incrementUserCounter(_ userCounter: UserCounterDataModel) -> AnyPublisher<RealmUserCounter, URLResponseError> {
```

Comment:
Hey @rachaelblue same comment here as in ```UserCountersAPI```.  I'd suggest changing the argument here to ```(id: String, increment: Int)``` or a type that encapsulates the attributes required for an increment.